src/styles/theme/global.js

```diff
@@ -1,6 +1,6 @@
 // 'polished/lib/mixins/normalize';
 import normalizeStyled from 'normalize-styled'
```

Comment:
I couldn't get this working when I cloned the repo. `npm` couldn't find `normalize-styled`. Polished comes with a normalize function, so `normalize-styled` shouldn't be necessary.